docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
   caddy:
     restart: on-failure
     networks: 
-      - service_net
+      - monitor_network
```

Comment:
You should add caddy to both service_net and monitor_net. By the way, I would recommend using the same naming scheme `xxx_net` for this section.

Reply:
Got it